src/blocks/danteSadeghi/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+    "block_name": "Dante's' Color Button",
```

Comment:
Isn't there a `'` too many in the block name? Shouldn't it be just `Dante's ...`

Reply:
Ah yes, slight typo. Just made the change! Thank you.